src/js/menu/menu-button.js

```diff
@@ -29,6 +29,14 @@ class MenuButton extends ClickableComponent {
 
     this.update();
 
+    /**
+     * Hide the menu if the number of items is less than or equal to this threshold. This defaults
+     * to 0 and whenever we add items to the menu we'll increment it.
+     *
+     * @type {Number}
+     */
+    this.hideThreshold_ = 0;
```

Comment:
this should probably move above `this.update()`. `createItems` gets called as part of `createMenu` which is called in `update`, so, we need to make sure that this threshold is initialized properly.